src/MobileUI/Pages/LeaderboardPage.xaml

```diff
@@ -21,18 +21,41 @@
           StatusBarStyle="LightContent" />
     </ContentPage.Behaviors>
     <Grid RowDefinitions="Auto,*">
-        <controls:SegmentedControl
-            Grid.Row="0"
-            Segments="{Binding Periods}"
-            HeightRequest="45"
-            Margin="15,10,15,15"
-            SelectedSegment="{Binding SelectedPeriod, Mode=OneWayToSource}">
-            <controls:SegmentedControl.Behaviors>
-                <mct:EventToCommandBehavior
-                    EventName="SelectionChanged"
-                    Command="{Binding FilterByPeriodCommand}" />
-            </controls:SegmentedControl.Behaviors>
-        </controls:SegmentedControl>
+        <Grid Grid.Row="0" ColumnDefinitions="*, Auto">
+            <controls:SegmentedControl
+                Grid.Column="0"
+                Segments="{Binding Periods}"
+                HeightRequest="45"
+                Margin="15,10,10,15"
+                SelectedSegment="{Binding SelectedPeriod, Mode=OneWayToSource}">
+                <controls:SegmentedControl.Behaviors>
+                    <mct:EventToCommandBehavior
+                        EventName="SelectionChanged"
+                        Command="{Binding FilterByPeriodCommand}" />
+                </controls:SegmentedControl.Behaviors>
+            </controls:SegmentedControl>
+
+            <Border
```

Comment:
Why is this not a `Button`?

Reply:
Mainly 2 reasons:
1. I had quite a few issues with buttons when I was migrating rewards to .NET MAUI. It was .NET 7 at that time and it wasn't that stable. I need to start using `Button` with .NET 8 though.

2. I just copied the code from `SegmentedControl` to make sure that the new button looks the same.